build-logic-commons/basics/src/main/kotlin/gradlebuild/basics/ImplementationCompletenessAttribute.kt

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2020 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package gradlebuild.basics
+
+import org.gradle.api.attributes.Attribute
+
+// TODO: Find a better name
```

Comment:
🤔 Should we find a better name? Is there a follow-up issue?

Reply:
How about now?

Reply:
What about ApiJarTypeAttribute or something like that?

Reply:
I am fine with the name, though the TODO should go away.

Reply:
I replaced the name but the push didn't make it. You can check now if you want.